tkm_circleCI_fieldTests/tkm_circleCI_fieldTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  tkm_circleCI_fieldTests.swift
+//  tkm_circleCI_field
+//
+//  Created by 長谷川 拓海(Takumi Hasegawa) on 2024/09/12.
+//  Copyright (c) 2024 *ReNKCHANNEL*. All rights reserved.
+//
+
+import XCTest
+
+final class tkm_circleCI_fieldTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        // Any test you write for XCTest can be annotated as throws and async.
+        // Mark your test throws to produce an unexpected failure when your test encounters an uncaught error.
+        // Mark your test async to allow awaiting for asynchronous code to complete. Check the results with assertions afterwards.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/TakumiHasegawa21/tkm_circleCI_field/blob/cf59bd640260b9f05f2b0309a0ec1fa6f4c32ad7/tkm_circleCI_fieldTests/tkm_circleCI_fieldTests.swift#L26"></span>Line should be 120 characters or less; currently it has 130 characters<br />
<code>line_length</code> <code>tkm_circleCI_fieldTests.swift:26</code></td>
    </tr>
  </tbody>
</table>